src/Modal.js

```diff
@@ -156,6 +160,19 @@ const Modal = React.createClass({
     if (this.props.keyboard && e.keyCode === 27) {
       this.props.onRequestHide();
     }
+  },
+
+  focusModalContent () {
+    this.lastFocus = document.activeElement;
```

Comment:
Is it connected with `iframes` ?
I'm just not aware of use cases for this.
(have no such expirience)